### ISSUE TITLE
Master authentication from non-domain device not working

### ISSUE THREAD

**Issue by WestfieldMX5** [veyon/veyon#904]
I have deployed veyon on a number of intune devices.  These devices are not domain joined, but the users logon with a domain account (UPN).
When I try to authenticate myself in the master, this works only when I'm doing this on a domain device.  When I try to authenticate from a non-domain joined device, I get a user / password error.  I have setup LDAP in the veyon configuration and tested that it works on the domain joined device.
On the non-domain joined device, I tested both the UPN as the DOMAIN\username from, both fail.
Basically, I have no way of monitoring the devices from my intune joined device.  Is there any way around it?

I'm using Veyon in a school environment, so all teachers much be able to login to the master of their classroom.  They too have a intune device (not domain joined).  I could create local accounts on each device but that's cumbersome because they'd possibly have different passwords for the local account and for the intune account.  It'll be a mess to maintain it.

**Comment by WestfieldMX5** [comment]
There's no point in trying to acces them from a non-domain device.  The classroom pc's are not visible in the domain either since they're intune managed.
